lib/theme-options/site-options.php

```diff
@@ -62,6 +62,12 @@
       'id'   => $prefix . 'og_fb_app_id',
       'type' => 'text',
     ),
+    array(
+      'name' => __( 'Google API Key', 'IGV' ),
```

Comment:
This is not a Metadata Option. It should have its own section at the top of the setting page